AstarteDeviceSDKCSharp/AstartePairingService.cs

```diff
@@ -52,20 +52,20 @@ public AstartePairingService(string pairingUrl, string astarteRealm)
         {
             List<AstarteTransport> transports = new();
             // Prepare the Pairing API request
-            HttpClient client = new();
-            client.DefaultRequestHeaders.Authorization = new AuthenticationHeaderValue("Bearer",
+            _httpClient.DefaultRequestHeaders.Authorization = new AuthenticationHeaderValue("Bearer",
             credentialSecret);
 
-            var response = await client
-                        .GetAsync(_pairingUrl + $"/{_astarteRealm}/devices/{deviceId}");
+            var response = await _httpClient.GetAsync(
+                _pairingUrl + $"/{_astarteRealm}/devices/{deviceId}");
 
             if (!response.IsSuccessStatusCode)
             {
                 throw new AstartePairingException(
                             "Request to Pairing API failed with "
                                 + response.StatusCode.ToString()
                                 + ". Returned body is "
-                                + response.Content.ToString());
+                                + (response.Content != null ?
+                                await response.Content.ReadAsStringAsync() : "empty"));
```

Comment:
What do you think to write this?
```suggestion
                                + (await response.Content?.ReadAsStringAsync() ?? "empty"));
```

Reply:
Since the `response.Content` is not nullable I made a new solution.

Reply:
So, since `response.Content` is not nullable, why do you check `Content != null`?